en/docs/learn/user-portal.md

```diff
@@ -0,0 +1,230 @@
+# User Portal Documentation
+
+## Introduction
+
+The new WSO2 Identity Server user portal is packed with a number of new 
+components through which users can manage their user account related 
+preferences with more convenience. The latest set of features that will be 
+available with the new user portal includes:
+
+- User profile management
+- Linked accounts
+- Export user profile
+- Reset password
+- Account recovery
+- Multi-factor authentication
+- Monitor active user sessions
+- Consent management
+- Review pending approvals
+
+In this section, we will give a brief explanation of what each of the 
+above-mentioned features are and we are going to guide you through how to use 
+these features in the user portal depending on your preferences and requirements.
+
+## Accessing the user portal and its components
+
+1.  Go to the dashboard URL: https://(host name):(port name)/user-portal/
```

Comment:
Access the WSO2 Identity Server User Portal at https://<HOST_NAME>:<PORT>/user-portal.